src/main/resources/api.yaml

```diff
@@ -30,6 +30,66 @@ paths:
             $ref: '#/definitions/LoginResult'
         400:
           description: Validation error
+  /projects:
+    get:
+      tags:
+        - projects
+      summary: Lists current user's projects
+      operationId: getProjects
+      produces:
+        - application/json
+      responses:
+        200:
+          description: Projects listed successfully
```

Comment:
`Projects retrieved successfully`?

Reply:
Czyli też `Retrieves current user's projects`? W sumie to jest listowanie zasobów

Reply:
`Retrieves a list of the current user's projects`?